packages/smart-wallet/src/smartWallet.js

```diff
@@ -239,6 +253,11 @@ export const prepareSmartWallet = (baggage, shared) => {
       /** @type {PublishKit<CurrentWalletRecord>} */
       currentPublishKit,
       walletStorageNode,
+      liveOffers: makeScalarBigMapStore('live offers', { durable: true }),
+      // Keep seats separate from the offers because we don't want to publish these.
+      liveOfferSeats: makeScalarBigMapStore('live offer seats', {
```

Comment:
I wonder about **scaling considerations** of adding these collections to wallet state. I don't see any consideration of it in the PR description. (In fact, I don't see any of the usual PR template.)

Do we ever iterate over all of them? If not, perhaps make it a weak map store?

Reply:
Good call to include in Scaling Considerations.

We do iterate over `liveOffers` to publish them.